code/modules/clothing/clothing.dm

```diff
@@ -1143,36 +1142,39 @@
 			. += "<span class='notice'><b>Alt-Shift-Click</b> to remove an accessory.</span>"
 	. += "<span class='notice'><b>Ctrl-Shift-Click</b> to roll down this jumpsuit.</span>"
 
+/// Suffix for jumpsuits used in .dmi files when rolled down
+#define JUMPSUIT_ROLLED_DOWN_SUFFIX "_d"
+
+/obj/item/clothing/under/proc/roll_undersuit(mob/living/carbon/human/user)
+	var/static/rolled_down_suffix_length = length(JUMPSUIT_ROLLED_DOWN_SUFFIX)
+	if(copytext(icon_state, -rolled_down_suffix_length) != JUMPSUIT_ROLLED_DOWN_SUFFIX)
```

Comment:
No need to overcomplicate this, please remove the static variable. Byond's bytecode will likely compile-time optimize it away anyways.